starkinfra/issuingtoken/__issuingtoken.py

```diff
@@ -0,0 +1,238 @@
+    """# Update IssuingToken entity
+    Update an IssuingToken by passing id.
+    ## Parameters (required):
+    - id [string]: IssuingToken id. ex: '5656565656565656'
+    ## Parameters (optional):
+    - status [string, default None]: You may block the IssuingToken by passing 'blocked' or activate by passing 'active' in the status. ex: "active", "blocked"
+    - tags [list of strings, default None]: list of strings for tagging. ex: ["travel", "food"]
+    - user [Organization/Project object, default None]: Organization or Project object. Not necessary if starkinfra.user was set before function call.
+    ## Return:
+    - target IssuingToken with updated attributes
+    """
+    payload = {
+        "status": status,
+        "tags": tags
+    }
+    return rest.patch_id(resource=_resource, id=id, payload=payload, user=user)    
+
+
+def cancel(id, user=None):
+    """# Cancel an IssuingToken entity
+    Cancel an entity previously created in the Stark Infra API by its ID
+    ## Parameters (required):
+    - id [string]: IssuingToken unique id. ex: "5656565656565656"
+    ## Parameters (optional):
+    - user [Organization/Project object, default None]: Organization or Project object. Not necessary if starkinfra.user was set before function call.
+    ## Return:
+    - canceled IssuingToken object
+    """
+    return rest.delete_id(resource=_resource, id=id, user=user)
+
+
+def parse(content, signature, user=None):
+
+    """# Create a single verified IssuingToken request from a content string
+    Use this method to parse and verify the authenticity of the request received at the informed endpoint.
+    Token requests are posted to your registered endpoint whenever IssuingTokens are received.
+    They present IssuingToken data that must be analyzed and answered with approval or declination.
+    If the provided digital signature does not check out with the StarkInfra public key, a stark.exception.InvalidSignatureException will be raised.
+    If the request is not answered within 2 seconds or is not answered with an HTTP status code 200 the IssuingToken will go through the pre-configured stand-in validation.
+    ## Parameters (required):
+    - content [string]: response content from request received at user endpoint (not parsed)
+    - signature [string]: base-64 digital signature received at response header "Digital-Signature"
+    ## Parameters (optional):
+    - user [Organization/Project object, default None]: Organization or Project object. Not necessary if starkinfra.user was set before function call
+    ## Return:
+    - Parsed IssuingToken object
+    """
+    return parse_and_verify(
+        content=content,
+        signature=signature,
+        user=user,
+        resource=_resource,
+        key="",
+    )
+
+
+def response_authorization(status, reason=None, activation_methods=None, design_id=None, tags=None):
+    """# Helps you respond IssuingToken authorization requests
```

Comment:
When a new tokenization is triggered by your user, a POST request will be made to your registered URL to get your decision to complete the tokenization.
The POST request must be answered in the following format, within 5 seconds, and with an HTTP status code 200.